### ISSUE TITLE
yarn test is broken in v1.2.0: Type error: '"functions/queries"' has no exported member named 'getMarketDepositsQuery'. Did you mean 'getDepositsQuery'?

### ISSUE THREAD

**Issue by arno01** [mars-protocol/interface#7]
Building with this Dockerfile https://github.com/mars-protocol/interface/pull/4

- `v1.1.0`: builds without issues
- `v1.2.0`: Type error: '"functions/queries"' has no exported member named 'getMarketDepositsQuery'. Did you mean 'getDepositsQuery'?

```
Step 7/22 : RUN yarn build
 ---> Running in 7749a6e18595
yarn run v1.22.19
$ yarn test && next build
$ jest
info  - Loaded env from /app/.env
warn  - Invalid next.config.js options detected: 
  - The value at .experimental has an unexpected property, images, which is not in the list of allowed properties (adjustFontFallbacks, adjustFontFallbacksWithSizeAdjust, allowedRevalidateHeaderKeys, amp, cpus, craCompat, disableOptimizedLoading, disablePostcssPresetEnv, esmExternals, appDir, externalDir, fallbackNodePolyfills, fetchCache, forceSwcTransforms, fullySpecified, gzipSize, incrementalCacheHandlerPath, isrFlushToDisk, isrMemoryCacheSize, largePageDataBytes, legacyBrowsers, manualClientBasePath, middlewarePrefetch, newNextLinkBehavior, nextScriptWorkers, optimizeCss, optimisticClientCache, outputFileTracingRoot, outputFileTracingIgnores, pageEnv, preCompiledNextServer, proxyTimeout, runtime, serverComponentsExternalPackages, scrollRestoration, sharedPool, sri, swcFileReading, swcMinify, swcMinifyDebugOptions, swcPlugins, swcTraceProfiling, urlImports, enableUndici, workerThreads, fontLoaders, webVitalsAttribution, mdxRs, turbopackLoaders, turbotrace).

See more info here: https://nextjs.org/docs/messages/invalid-next-config
warn  - You have enabled experimental feature (images) in next.config.js.
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.

warn  - Invalid next.config.js options detected: 
  - The value at .experimental has an unexpected property, images, which is not in the list of allowed properties (adjustFontFallbacks, adjustFontFallbacksWithSizeAdjust, allowedRevalidateHeaderKeys, amp, cpus, craCompat, disableOptimizedLoading, disablePostcssPresetEnv, esmExternals, appDir, externalDir, fallbackNodePolyfills, fetchCache, forceSwcTransforms, fullySpecified, gzipSize, incrementalCacheHandlerPath, isrFlushToDisk, isrMemoryCacheSize, largePageDataBytes, legacyBrowsers, manualClientBasePath, middlewarePrefetch, newNextLinkBehavior, nextScriptWorkers, optimizeCss, optimisticClientCache, outputFileTracingRoot, outputFileTracingIgnores, pageEnv, preCompiledNextServer, proxyTimeout, runtime, serverComponentsExternalPackages, scrollRestoration, sharedPool, sri, swcFileReading, swcMinify, swcMinifyDebugOptions, swcPlugins, swcTraceProfiling, urlImports, enableUndici, workerThreads, fontLoaders, webVitalsAttribution, mdxRs, turbopackLoaders, turbotrace).

See more info here: https://nextjs.org/docs/messages/invalid-next-config
PASS src/functions/redbank/maxBorrowableAmount.test.ts
PASS src/functions/redbank/produceUpdatedAssetData.test.tsx
PASS src/functions/fields/test/numberFunctions.test.ts

Test Suites: 3 passed, 3 total
Tests:       25 passed, 25 total
Snapshots:   0 total
Time:        0.995 s
Ran all test suites.
info  - Loaded env from /app/.env
warn  - Invalid next.config.js options detected: 
  - The value at .experimental has an unexpected property, images, which is not in the list of allowed properties (adjustFontFallbacks, adjustFontFallbacksWithSizeAdjust, allowedRevalidateHeaderKeys, amp, cpus, craCompat, disableOptimizedLoading, disablePostcssPresetEnv, esmExternals, appDir, externalDir, fallbackNodePolyfills, fetchCache, forceSwcTransforms, fullySpecified, gzipSize, incrementalCacheHandlerPath, isrFlushToDisk, isrMemoryCacheSize, largePageDataBytes, legacyBrowsers, manualClientBasePath, middlewarePrefetch, newNextLinkBehavior, nextScriptWorkers, optimizeCss, optimisticClientCache, outputFileTracingRoot, outputFileTracingIgnores, pageEnv, preCompiledNextServer, proxyTimeout, runtime, serverComponentsExternalPackages, scrollRestoration, sharedPool, sri, swcFileReading, swcMinify, swcMinifyDebugOptions, swcPlugins, swcTraceProfiling, urlImports, enableUndici, workerThreads, fontLoaders, webVitalsAttribution, mdxRs, turbopackLoaders, turbotrace).

See more info here: https://nextjs.org/docs/messages/invalid-next-config
warn  - You have enabled experimental feature (images) in next.config.js.
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.

Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

info  - Linting and checking validity of types...
Failed to compile.

./src/hooks/queries/useMarketDeposits.tsx:2:10
Type error: '"functions/queries"' has no exported member named 'getMarketDepositsQuery'. Did you mean 'getDepositsQuery'?

  1 | import { useQuery } from '@tanstack/react-query'
> 2 | import { getMarketDepositsQuery } from 'functions/queries'
    |          ^
  3 | import { gql, request } from 'graphql-request'
  4 | import { useEffect } from 'react'
  5 | import useStore from 'store'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c yarn build' returned a non-zero code: 1
```

**Comment by arno01** [comment]
I've disabled the checking and deployed the v1.2.0 here https://mars.akash.pro

```
interface$ git diff next.config.js package.json
diff --git a/next.config.js b/next.config.js
index e9f9259..4cdff5f 100644
--- a/next.config.js
+++ b/next.config.js
@@ -47,6 +47,23 @@ const moduleExports = {
       },
     ]
   },
+  sentry: {
+    disableServerWebpackPlugin: true,
+    disableClientWebpackPlugin: true,
+  },
+  output: 'standalone',
+  typescript: {
+    // !! WARN !!
+    // Dangerously allow production builds to successfully complete even if
+    // your project has type errors.
+    // !! WARN !!
+    ignoreBuildErrors: true,
+  },
+  // eslint: {
+  //   // Warning: This allows production builds to successfully complete even if
+  //   // your project has ESLint errors.
+  //   ignoreDuringBuilds: true,
+  // },
 }
 
 module.exports = moduleExports
diff --git a/package.json b/package.json
index 31e7006..8419ef7 100644
--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "license": "SEE LICENSE IN LICENSE FILE",
   "scripts": {
     "dev": "next dev",
-    "build": "yarn test && next build",
+    "build": "next build",
     "export": "next export",
     "start": "next start",
     "lint": "eslint ./src/ && yarn prettier-check",
```
$ git checkout package.json; (rm -f package.json && jq -r '(.scripts.build="next build")' > package.json) < package.json; git diff
Updated 0 paths from the index
diff --git a/package.json b/package.json
index 31e7006..8419ef7 100644
--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "license": "SEE LICENSE IN LICENSE FILE",
   "scripts": {
     "dev": "next dev",
-    "build": "yarn test && next build",
+    "build": "next build",
     "export": "next export",
     "start": "next start",
     "lint": "eslint ./src/ && yarn prettier-check",

$ git fetch --tags upstream; TAG=$(git describe --tags --abbrev=0); docker build -t andrey01/mars-fe:${TAG} . && docker push andrey01/mars-fe:${TAG}
```

**Comment by linkielink** [comment]
This should be fixed with `v1.2.1`
